src/Umbraco.Web.UI.Client/src/common/resources/elementtype.resource.js

```diff
@@ -0,0 +1,31 @@
+/**
+    * @ngdoc service
+    * @name umbraco.resources.elementTypeResource
+    * @description Loads in data for element types
+    **/
+function elementTypeResource($q, $http, umbRequestHelper) {
+
+    return {
+
+        getAll: function () {
+
+            var url = Umbraco.Sys.ServerVariables.umbracoSettings.umbracoPath + "/backoffice/UmbracoApi/ElementType/GetAll";
```

Comment:
We have an API for resolving paths correctly which is used by all other resources so we should follow the exact same thing we do everywhere else: umbRequestHelper.getApiUrl

Reply:
I'll have @warrenbuckley making sure this is done accordingly. :-)

Reply:
Yep I will fix & ensure the APIController is registered in Umbraco.Sys.ServerVariables

Reply:
@Shazwazza & @nielslyngsoe this was fixed in this commit here
https://github.com/umbraco/Umbraco-CMS/pull/8273/commits/c53ea9382f50216a581aefa754598f5b8adc6dee

So I will mark this conversation as resolved